script/parsers/manifestparser.js

```diff
@@ -29,18 +31,23 @@ define('bigscreenplayer/parsers/manifestparser',
          } else {
            return {error: 'Error parsing DASH manifest attributes'};
          }
+         DebugTool.info('Done parsing MPD');
```

Comment:
A lot of these logs you should probably remove... errors are fine to leave in. I am assuming you just used them for testing purposes?

Reply:
Removed the debug tool require and extra logging statements.